.chloggen/elasticsearchexporter_log_body.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
The `otel` mapping mode used to be unstable, but that warning has been [removed](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37240) in the latest version, v0.118.0. This change is now a breaking change in a Beta stability component. What are the consequences to users upgrading the collector? Will this cause the data to be rejected by Elasticsearch? Can this be somehow alleviated by the user?

Reply:
Data will still be ingested, no rejections. It will just be mapped differently. The field `body.structured` already exists, it will just be used for all kinds of bodies.

Reply:
This feels in line with the expectations set by the beta stability level.

> Same as Alpha, but the configuration options are deemed stable. While there might be breaking changes between releases, component owners should try to minimize them.

The config options are unchanged and you'll still be able to ingest data. The field in ES is just named slightly differently. WDYT?

Reply:
I was under the impression that breaking changes on behavior are fine, while breaking changes on config should follow the policy.